change/@ni-eslint-config-playwright-1125d689-3a37-4a84-b8ef-19749c7339f6.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
@rajsite had a good suggestion to update our playwright ruleset docs to remind clients to only apply it to playwright code, since we don't want this rule suppression to affect other code

Reply:
I took a stab at this. Feel free to suggest alternate language.

Reply:
Language is good, question is if the configuration snippet should show using the rules in an override that targets playwright tests (with a comment to cover things that might be application specific such as test file name patterns).

Reply:
This fell way off the radar somehow but sounds good 👍

Reply:
Feel free to merge when ready

Reply:
Actually it just loosens a rule so we can merge but PR description says there are follow-ups for #123